reference/3.0/Microsoft.PowerShell.Core/About/about_Command_Precedence.md

```diff
@@ -40,16 +39,75 @@ following rules.
   path to the script file.
 
   To run a script that is in the current directory, specify the full path, or
-  type a dot `.` to represent the current directory.
+  type a dot `.\` to represent the current directory.
```

Comment:
Should the be "type a dot and backslash" instead?

Reply:
fixed